migrations/20180325204028_remove-unused-ghost-tables.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+
+exports.up = async knex => {
```

Comment:
ooo fancy Async

Reply:
That's it? :P. Such a useful review haha

Reply:
Slowly reading during networks 😆. My single comments actually look quite annoying on Github UI though...

Reply:
Oh yeah I don't think I remembered that it used the `User reviewed` wording in the single comment case either